src/store/index.js

```diff
@@ -1,48 +1,82 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
-import { app } from '../firebaseConfig'
+import { app, db } from '../firebaseConfig'
 import { getAuth, createUserWithEmailAndPassword, signInWithEmailAndPassword } from "firebase/auth";
+import { doc, setDoc, getDoc } from "firebase/firestore";
+import router from '../router';
+
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
     user: {
       name: '',
+      balance: '',
     },
     status: false
   },
+  getters:{
+    user: state => state.user,
+  },
   mutations: {
-    setUserName(state, userName) {
-      state.user.name = userName;
-      console.log(state.user.name);
-    },
     onAuthStateChanged(state, status) {
       state.status = status;
+    },
+    setUserData(state, {name, balance}) {
+      const user = {
+        name: name,
```

Comment:
オブジェクトのkeyとvalueの名前が一致している場合、以下のように省略記法が使用できますので、修正してみましょう。
```suggestion
        name,
```